src/pathOrderOptimizer.cpp

```diff
@@ -188,14 +188,16 @@ void LineOrderOptimizer::optimize()
         int best_line_idx = -1;
         float best_score = std::numeric_limits<float>::infinity(); // distance score for the best next line
 
-        for(unsigned int close_line_poly_idx :  line_bucket_grid.findNearbyObjects(prev_point)) /// check if single-line-polygon is close to last point
+        /// check if single-line-polygon is close to last point
+        for(typename SparseGrid<unsigned int>::Elem close_line_elem :
+                line_bucket_grid.getNearby(prev_point, gridSize))
```

Comment:
I think you should just use getNearbyVals here, since you introduced a function for that.